src/encoded/static/components/dataset.js

```diff
@@ -2511,10 +2511,13 @@ SeriesDatasetTableSection.defaultProps = {
  */
 const SeriesExperimentTable = ({ context, experiments, title, tableColumns, sortColumn, adminUser, accessLevel }) => {
     if (experiments.length > 0) {
-        // Get all the control experiments from the given experiments' `possible_controls`. Then
+        // Get all the control experiments from the given experiments' `possible_controls`, `elements_mappings` and `elements_cloning`. Then
         // filter those out of the given experiments.
         const controls = _.uniq(experiments.reduce((accControls, experiment) => (experiment.possible_controls ? accControls.concat(experiment.possible_controls) : accControls), []), (control) => control['@id']);
-        const experimentsWithoutControls = experiments.filter((experiment) => !controls.find((control) => control['@id'] === experiment['@id']));
+        const elementsMappings = _.uniq(experiments.reduce((accMappings, experiment) => (experiment.elements_mappings ? accMappings.concat(experiment.elements_mappings) : accMappings), []), (mappings) => mappings['@id']);
```

Comment:
It is best to use in-built Javascript functionality over third party libraries. Javascript has Set which does what underscore.js uniq() does.